google/cloud/security/common/data_access/dao.py

```diff
@@ -97,6 +97,30 @@ def load_data(self, resource_name, timestamp, data):
                 OperationalError, ProgrammingError) as e:
             raise MySQLError(resource_name, e)
 
+    def select_record_count(self, resource_name, timestamp):
+        """Select the record count from a snapshot table.
+
+        Args:
+            resource_name: String of the resource name, which is embedded in
+                the table name.
+            timestamp: String of timestamp, formatted as YYYYMMDDTHHMMSSZ.
+
+        Returns:
+             Integer of the record count in a snapshot table.
+
+        Raises:
+            MySQLError: An error with MySQL has occurred.
+        """
+        try:
+            record_count_sql = select_data.RECORD_COUNT.format(
+                resource_name, timestamp)
+            cursor = self.conn.cursor()
+            cursor.execute(record_count_sql)
```

Comment:
Great catch.  I tried it, however not working.  When I look up the error, it seems that parameterized query only works for column value, and not anything else.
http://stackoverflow.com/a/12831519/2830207